deps/rabbitmq_mqtt/src/rabbit_mqtt_processor.erl

```diff
@@ -192,7 +192,8 @@ process_connect(
                        'Maximum-Packet-Size' => persistent_term:get(
                                                   ?PERSISTENT_TERM_MAX_PACKET_SIZE_AUTHENTICATED),
                        'Subscription-Identifier-Available' => 0,
-                       'Shared-Subscription-Available' => 0
+                       'Shared-Subscription-Available' => 0,
+                       'Wildcard-Subscription-Available' => 0
```

Comment:
We should revert this commit because RabbitMQ MQTT does support wildcard subscriptions.

Reply:
@ChunyiLyu I dropped your commit `Return wildcard sub unsupport in connack` when rebasing. I pushed a backup to branch https://github.com/ansd/rabbitmq-server/tree/mqtt5-backup-2023-03-15 such that your commit is not lost in case you will need it.